hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1814,7 +1889,7 @@ private AbfsPerfInfo startTracking(String callerName, String calleeName) {
    * in a LIST or HEAD request.
    * The etag is included in the java serialization.
    */
-  private static final class VersionedFileStatus extends FileStatus
+  static final class VersionedFileStatus extends FileStatus
```

Comment:
why are we removing the access modifier?

Reply:
The class is still package-protected. Have removed private as there is an addition of assertion on the values of VersionedFileStatus objects in ITestAbfsCustomEncryption. The constructor to this class is still private -> so only AzureBlobFileSystemStore(parent class of VersionedFileStatus) can create object of this class.